recipes-bsp/hm-interface-names/hm-interface-names_1.1.bb

```diff
@@ -1,10 +1,11 @@
-DESCRIPTION = "udev rule and script for persistent names of CAN and Ethernet"
+SUMMARY = "Service and script for persistent names of CAN and Ethernet"
+DESCRIPTION = "A service that runs script for reordering network and CAN interfaces to keep the canX and ethX names but order them in the same order regardless of which gets created first by the drivers "
+
 SECTION = "udev"
 LICENSE = "CLOSED"
 SRC_URI = "\
-	file://10-hm-interface-names.rules \
-	file://hm-interface-names.service \
-	"
+    file://hm-interface-names.service \
+"
```

Comment:
the file://10-hm-interface-names.rules \ is never used. maybe a mistake to not include it?

Reply:
The rule should not be needed anymore, the script waits until all interfaces are up or timeout.